### PR TITLE
Create gathering scripts

### PR DESCRIPTION
**Pull request by nelsonand**
## A working log for creation of gathering scripts ##

### CODE REVIEW

**Comment by nelsonand** [comment]
@joshuakcrane https://github.com/nelsonand/GARDN-M/pull/3 looks awesome. Feel free just to add future scripts like that (if you feel like writing more) directly to this branch. 

Eventually, once all (or a good fraction) of the sites are supported, we'll make sure they are all called by [main_gather.py](https://github.com/nelsonand/GARDN-M/blob/main/scripts/gather/main_gather.py)

Regarding saving the data - I just realized the original sheet I uploaded was corrupted, to that is fixed now by [a332488](https://github.com/nelsonand/GARDN-M/pull/1/commits/a332488e0a9226a78d53de82d613dbe6aeac50dd) I'm not sure if it makes more sense to put everything into one file like this, or to store each site in a separate .csv. I guess both would be fine. 

A note on the data in [20230320_alldata.xlsx](https://github.com/nelsonand/GARDN-M/blob/make_gather/data/processed_data/20230320_alldata.xlsx). You'll notice that in addition to the rank, we also listed 'rank by gap' for each of the categories, and a "comp score." The rank is most important for now, so I think that's all we need for the moment. The other stuff can be added later. The "comp score" is calculated... so this should probably not actually be stored in the raw data but rather be part of the analysis scripts. I'll work on that soon hopefully. 

Long story short - https://github.com/nelsonand/GARDN-M/pull/3 is perfect and exactly awesome and perfect.

**Comment by nelsonand** [comment]
Thinking more on this, I kinda like the idea of each website having it's own csv file... let's start there.
